### ISSUE TITLE
Run wl-copy asynchronously 

### ISSUE THREAD

**Issue by nonetrix** [Satty-org/Satty#129]
When you set the copy command to `wl-copy` it will copy as expected, however, it will also freeze Satty until it is done copying. You won't really notice this much, besides the fact that the copy button will freeze it's animation however. I tried changing the command to `wl-copy &` but this didn't seem to work

**Comment by gabm** [comment]
yep, adding `&` is a "shell" feature. That doesn't work in Satty. 

We would need to implement this in Satty itself. While I am open for that, it adds additional complication: you'd need to wait until the command has completed, before shutting down Satty - otherwise you end up with nothing (valid) in the clipboard
